tools/archives/pyvo_integration/astronomical_archives.xml

```diff
@@ -0,0 +1,1023 @@
+                    <option value="custom">Custom (Input data product type yourself)</option>
+                    <option value="image">Image</option>
+                    <option value="cube">Cube</option>
+                    <option value="dynamic-spectrum">Dynamic spectrum</option>
+                    <option value="event-list">Event list</option>
+                    <option value="light-curve">Light curve</option>
+                    <option value="measurements">Measurements</option>
+                    <option value="polarization-cube">Polarization cube</option>
+                    <option value="polarization-resolved-dataset">Polarization resolved dataset</option>
+                    <option value="polarized-spectrum">Polarized spectrum</option>
+                    <option value="sed">SED</option>
+                    <option value="slit-spectrum">Slit spectrum</option>
+                    <option value="spatially-resolved-dataset">Spatially resolved dataset</option>
+                    <option value="spectral-cube">Spectral cube</option>
+                    <option value="spectrally-resolved-dataset">Spectrally resolved dataset</option>
+                    <option value="spectrum">Spectrum</option>
+                    <option value="temporally-resolved-dataset">Temporally resolved dataset</option>
+                    <option value="time-cube">Time cube</option>
+                    <option value="timeseries">Time series</option>
+                    <option value="velocity-curve">Velocity curve</option>
+                    <option value="visibility">Visibility</option>
+                  </param>
+                </when>
+                <when value="custom">
+                  <param name="dataproduct_type" type="text" label="Data product type" help="Logical data product type" />
+                </when>
+              </conditional>
+              <param name="obs_collection" type="text" label="Collection" help="Name of the observation collection" />
+              <param name="obs_title" type="text" label="Observation title (archive specific)" />
+              <param name="obs_id" type="text" label="Observation Id" />
+              <param name="facility_name" type="text" label="Facility name" help="Name of the facility used for this observation" />
+              <param name="instrument_name" type="text" label="Instrument name" help="Name of the instrument used for this observation" />
+              <param name="em_min" type="text" label="Start of the energy range, vacuum wavelength in meters" />
+              <param name="em_max" type="text" label="Stop of the energy range, vacuum wavelength in meters" />
+              <param name="obs_publisher_id" type="text" label="Publisher dataset ID" />
+              <param name="s_fov" type="text" label="Diameter (bounds) of the covered region (deg)" />
+              <param name="calibration_level" type="select" label="Calibration level (0, 1, 2, 3, 4, 5)" >
```

Comment:
why not an integer with optional="true" and min=0 and max=5? It would give you a nice slider.